x-pack/legacy/plugins/siem/public/components/charts/areachart.test.tsx

```diff
@@ -310,7 +292,7 @@ describe('AreaChartWithCustomPrompt', () => {
         },
       ],
     ],
-  ])('renders prompt', (data: ChartConfigsData[] | [] | null | undefined) => {
```

Comment:
@elastic/siem this table does not match the type definition, I'm also not 100% sure jest is running this the way it was intended, or what. It's passing as it is, but I'd really appreciate if someone could take another look at this and make sure the table works when it's defined like https://github.com/elastic/kibana/blob/00cfd1193dd265cb6e69e33534c3240430bdfaf4/x-pack/legacy/plugins/siem/public/components/charts/barchart.test.tsx#L212 (it doesn't right now)

Reply:
Thank you @spalger for the clean up! I think it would be clearer if we rename the `table` into `tables`, as in this description I was trying to list all the cases that `data` passed into `BarChartWithCustomPrompt` render only `ChartHolder`.

1. data = undefined
2. data = null
3. ``` 
    data = [
      { key: 'uniqueSourceIps', color: '#DB1374' },
      {
        key: 'uniqueDestinationIps',
        color: '#490092',
      },
    ]
 ```
...etc

So I used https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout, I thought I had to wrap data with array to map the pattern, maybe that's what makes it confused. Let me try if I can flatten that.

Reply:
@angorayc I'm merging master now, you might find it helpful to print the arguments in the test name by including `%j`, like `'renders prompt when chart data = %j'`.

Reply:
Tried to flatten the param that feed in to the component, but some of cases failed accordingly. I printed out the `data` props from `barchart.tsx` but the data seems to remain the same when it is not flatten.

I am fine with the change, after this PR got merged, I'll raise another one to update the the description so it will print which dataset it's testing and also following up if I can simplify the input.

Reply:
Thanks again for taking care of the update @spalger